IbisPlugins/GPU_VolumeReconstruction/gpu_volumereconstruction.cpp

```diff
@@ -1,37 +1,27 @@
 #include "gpu_volumereconstruction.h"
 #include "ibisitkvtkconverter.h"
-#include "scenemanager.h"
-#include "sceneobject.h"
-#include "usacquisitionobject.h"
-#include "vtkTransform.h"
-#include "vtkLinearTransform.h"
 #include "vtkMatrix4x4.h"
 
 #include "vtkImageData.h"
-#include "vtkImageShiftScale.h"
-#include "vtkImageLuminance.h"
-#include "vtkMath.h"
-
-#include "imageobject.h"
-
 
 GPU_VolumeReconstruction::GPU_VolumeReconstruction()
 {
 }
 
 GPU_VolumeReconstruction::~GPU_VolumeReconstruction()
 {
-
 }
 
 void GPU_VolumeReconstruction::CreateReconstructor()
 {
+    m_mutex.lock();
```

Comment:
I don't get the use of this mutex. It only seems to prevent that two different threads call CreateReconstructor at the same time on the same instance of GPU_VolumeReconstruction.